staging_service/autodetect/GenerateMappings.py

```diff
@@ -120,7 +129,7 @@
     fba_model_id,
     import_specification,
 ]
-file_format_to_app_mapping[JSON] = [escher_map_id]
+file_format_to_app_mapping[JSON] = [escher_map_id, import_specification]
```

Comment:
Yeah, I thought of that. I guess that would be a trivial change, but I'm not sure the best thing to call it. I also wonder if it would be confusing to users at all? They'd have to read instructions on the whole DTS process anyway.

Reply:
Oh, I take it back, that's not trivial at all. It means a narrative change at minimum, possibly an API change if we want to be really thorough. Maybe a new API endpoint?

Reply:
I would make it a query param probably

Reply:
Just checking here - you're saying the separate type and query param is the way you're going to go?

Reply:
Also, if it's going to work that way, there probably shouldn't be a JSON: dts_parser entry in the parser mapping

Reply:
This is how it work right now, as of this PR, and everything else that's in it. A later PR that updates the endpoint to use a query param will probably undo this. Also, the Narrative will need to have a mapping of some sort, which is what this sets up. So it might not be from `.json` -> `Import Specification`, but it'll need to be something similar.

I don't want this to get more complicated and would like to keep that in a single PR with the `bulk_specification` endpoint change.

Reply:
Fair enough, as long as the plan is clear